### PR TITLE
ice and fire terminal + activatable door

### PR DESCRIPTION
**Pull request by PyroPoro**

### CODE REVIEW

**Comment by AlekseyPanas** [comment]
Task linked: [CU-866b5mg07 Terminal VFX](https://app.clickup.com/t/866b5mg07)

**Review by AlekseyPanas — APPROVED** [review]
Looks good

File: Assets/Prefabs/InteractableObjects/Terminals/FireTerminal/FireTerminal.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class FireTerminal : ATerminal<TemperatureEffect>
+{
+    [SerializeField] private float threshold;
+    public override bool IsAboveThreshold(TemperatureEffect effect)
+    {
+        if (effect.TempDelta >= threshold){
+            return true;
+        }
+        return false;
```
Comment (1whatleytay) [inline Assets/Prefabs/InteractableObjects/Terminals/FireTerminal/FireTerminal.cs:13]: ?

```suggestion
        return effect.TempDelta >= threshold;
```

File: Assets/Prefabs/InteractableObjects/Terminals/IceTerminal/IceTerminal.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+public class IceTerminal : ATerminal<TemperatureEffect>
+{
+    [SerializeField] private float threshold;
+    public override bool IsAboveThreshold(TemperatureEffect effect)
+    {
+        if (effect.TempDelta <= threshold){
+            return true;
+        }
+        return false;
```
Comment (1whatleytay) [inline Assets/Prefabs/InteractableObjects/Terminals/IceTerminal/IceTerminal.cs:13]: same

File: Assets/Scripts/Barriers/ABarrierActivatable.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+using System;
+using UnityEngine;
+using Unity.Netcode;
+using System.Collections.Generic;
+
+public abstract class ABarrierActivatable : NetworkBehaviour {
+    [SerializeField] List<GameObject> activatables;
```
Comment (1whatleytay) [inline Assets/Scripts/Barriers/ABarrierActivatable.cs:7]: Couldn't you just make this a list of `AActivatable`?

**Review by 1whatleytay — DISMISSED** [review]
Some comments, but not blocking. Good stuff

**Review by AlekseyPanas — APPROVED** [review]

**Review by 1whatleytay — APPROVED** [review]

**Pull request merged**